### DISCUSSION TITLE
Accurate depth maps from stero images

### DISCUSSION THREAD

**Discussion by CounterFlow64** [alicevision/AliceVision#1656] [Ideas]
https://replicate.com/arielreplicate/crestereo

https://github.com/megvii-research/CREStereo

This open source AI creates staggeringly accurate depth maps from just two images from slightly different angles. And it is rather quick. Could this be used to significantly speed up and improve depth map estimation for photogrammetry?
![image](https://github.com/alicevision/AliceVision/assets/79510944/7b5a3094-8693-4fb9-bc99-be41f79d666d)

**Comment by natowi** [comment]
Generally: yes. However adding deep learning to Meshroom is not easy. There needs to be a way to manage the framework and models. It can introduce additional complexity in the build process and could drastically increase the requirements.
https://github.com/alicevision/meshroom/issues/528

There are some first Nerf bridge nodes https://github.com/joreeves/mr2nerf and even first concepts for deep feature extraction https://github.com/PIX3LFLUX/MeshroomDFM.
So yes, give it some time and Meshroom will likely support something in this direction at some point :)
